statistics-dashboard-rest/src/main/java/statistics/dashboard/rest/config/StatisticsRestApplication.java

```diff
@@ -0,0 +1,144 @@
+import springfox.documentation.builders.PathSelectors;
+import springfox.documentation.builders.RequestHandlerSelectors;
+import springfox.documentation.service.ApiInfo;
+import springfox.documentation.service.Contact;
+import springfox.documentation.spi.DocumentationType;
+import springfox.documentation.spring.web.plugins.Docket;
+import springfox.documentation.swagger2.annotations.EnableSwagger2;
+
+/**
+ * The web application making available the repository functionality. This provides all the
+ * configuration and is the starting point for all injections and beans. It also performs the
+ * required setup.
+ */
+@Configuration
+@EnableWebMvc
+@EnableSwagger2
+@ComponentScan(basePackages = {"statistics.dashboard.rest"})
+public class StatisticsRestApplication implements WebMvcConfigurer, InitializingBean {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(StatisticsRestApplication.class);
+
+  private final ApplicationProperties properties;
+
+  private MongoClient mongoClientForEntities;
+
+  /**
+   * Constructor.
+   *
+   * @param properties The properties.
+   */
+  @Autowired
+  public StatisticsRestApplication(ApplicationProperties properties) {
+    this.properties = properties;
+  }
+
+  @Override
+  public void afterPropertiesSet() {
+    System.out.println("Web application initializing");
+  }
+
+//
+//    // Set the SOCKS proxy
+//    if (properties.isSocksProxyEnabled()) {
+//      new SocksProxy(properties.getSocksProxyHost(), properties.getSocksProxyPort(),
+//              properties.getSocksProxyUsername(), properties.getSocksProxyPassword()).init();
+//    }
+//
+//    // Set the truststore.
+//    LOGGER.info("Append default truststore with custom truststore");
+//    if (StringUtils.isNotEmpty(properties.getTruststorePath())
+//            && StringUtils.isNotEmpty(properties.getTruststorePassword())) {
+//      CustomTruststoreAppender.appendCustomTrustoreToDefault(properties.getTruststorePath(),
+//              properties.getTruststorePassword());
+//    }
+//
+//    // Create the mongo connection
+//    LOGGER.info("Creating Mongo connection");
+//    final MongoProperties<IllegalArgumentException> mongoProperties = new MongoProperties<>(
+//            IllegalArgumentException::new);
+//    mongoProperties
+//            .setAllProperties(properties.getMongoHosts(), new int[]{properties.getMongoPort()},
+//                    properties.getMongoAuthenticationDb(), properties.getMongoUsername(),
+//                    properties.getMongoPassword(), properties.isMongoEnableSsl(),
+//                    ReadPreferenceValue.PRIMARY_PREFERRED, properties.getMongoApplicationName());
+//    mongoClientForEntities = new MongoClientProvider<>(mongoProperties).createMongoClient();
+//  }
+
+  @Bean(name = DispatcherServlet.MULTIPART_RESOLVER_BEAN_NAME)
+  public StandardServletMultipartResolver getMultipartResolver() {
+    return new StandardServletMultipartResolver();
+  }
```

Comment:
I don't think we need this method. We're not doing multipart form data.

Reply:
Just tried to start the application without this method, and it seems that it is not needed, just removed it